src/fatality.js

```diff
@@ -6,5 +6,5 @@ module.exports = err => {
   log.error(err);
 
   lockfile.release()
-    .then(() => process.exit(1));
+    .then(() => process.exit(0));
```

Comment:
This looks wrong. Why are we exiting with success? Needs a comment at least...

Reply:
Sorry, accidental commit.

There is currently a bug where, if I close horti manually (with ctrl+c) it and it's children (api and sentinel) die, but if horti crashes they do not. This was an attempt to see if it was exit code related, but apparently not. I'll revert it.